olive/passes/onnx/conversion.py

```diff
@@ -65,6 +65,7 @@ def _convert_model_on_device(
         output_model_path: str,
         device: str,
     ):
+        model.extend_model_attributes_from_hf_config()
```

Comment:
>extend_model_attributes_from_hf_config [](http://example.com/codeflow?start=14&length=38)

could we call it in pass.run?

Reply:
![image](https://github.com/microsoft/Olive/assets/13343117/c9fcd01d-4286-4131-b0af-69a6ec6d7ab7)
Like this?

Reply:
yes